www/src/components/banner.js

```diff
@@ -2,14 +2,14 @@ import React from "react"
 import styled from "@emotion/styled"
 import { OutboundLink } from "gatsby-plugin-google-analytics"
 
-import { colors, space, dimensions, fonts } from "../utils/presets"
+import { colors, space, sizes, fonts } from "../utils/presets"
 
 const horizontalPadding = space[6]
 const backgroundColor = colors.gatsby
 
 const BannerContainer = styled(`aside`)`
   background-color: ${backgroundColor};
-  height: ${dimensions.bannerHeight};
+  height: ${sizes.bannerHeight};
```

Comment:
Yeah, I definitely think having `sizes` makes a lot of sense

Reply:
💯 Next up: Tidying up `breakpoints`, and adding `mediaQueries`!
Thanks so much for that initiative @jxnblk! 🙏